inc/formanswer.class.php

```diff
@@ -41,6 +41,12 @@ class PluginFormcreatorFormAnswer extends CommonDBTM
 
    const SOPTION_ANSWER = 900000;
 
+   // Values choosen to not conflict with status of ticket constants
+   // @see PluginFormcreatorIssue::getNewStatusArray
```

Comment:
Need harmonization with GLPI

ideas : 
- have similar constants in CommonITILObject
- convert formanswers into something inherited from ComonITILObject (seems complicated) ?